apps/site/lib/site_web/templates/event/_popup.html.eex

```diff
@@ -0,0 +1,35 @@
+<%#
+  Uses the 3rd-party a11y-dialog library as a base to create
+  accessible dialog windows for various use cases.
+  See documentation at https://a11y-dialog.netlify.app
+%>
+<div
+  class="m-event-overlay <%= @class %>"
+  id="<%= @id %>"
+  aria-label="Event summary for <%= @event.title %>"
+  aria-hidden="true"
+>
+  <div role="document">
+    <div class="u-flex-container">
+      <div class="m-event__title">
+        <%= event_title_link(@conn, @event) %>
+      </div>
+
+      <button type="button" data-a11y-dialog-hide aria-label="Close dialog">
+        &times;
+      </button>
+    </div>
+
+    <div class="u-flex-container">
+      <%= render "_add_button.html",
+          range: %{start: @event.date, stop: @event.date_end},
+          conn: @conn,
+          class: "popup-version",
+          path: @event.path %>
+      <div class="u-flex-one">
+        <strong>Tuesday, January 5, 2021</strong>
+        <div class="m-event__date-range">6 PM - 8 PM</div>
```

Comment:
These are hard-coded 😱  I assume you meant to read from the actual event?

Reply:
OH HA!  That was from the previous round of work, so I totally looked over it